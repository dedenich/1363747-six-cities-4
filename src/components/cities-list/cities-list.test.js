import React from 'react';
import CitiesList from './cities-list.jsx';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import configureStore from "redux-mock-store";

import mockStoreConf from '../../mocks/mock-store-conf.js';

const mockStore = configureStore([]);

it(`renders correctly`, () => {
  const store = mockStore(mockStoreConf);

  const tree = renderer.create(
      <Provider store={store}>
        <CitiesList/>
      </Provider>
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
