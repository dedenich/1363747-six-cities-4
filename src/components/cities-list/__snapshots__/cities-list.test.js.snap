// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    
  </ul>
</section>
`;
